docs/manualGrading/index.md

```diff
@@ -94,10 +94,61 @@ The user interface for manual grading is built to encourage a workflow where all
+
+- If it is set to manual points, the score obtained by the rubric grading is added to the points from the auto grader to result in the final score. This type of grading is often used for questions where the manual grading complements the auto grading, such as in grading rationales for specific calculations, or for code style in programming questions.
+- If the rubric is set to total points, the auto grader component is ignored, and the rubric grading overrides the final score of the question as a whole. This setting is used when the auto grading method is unable to capture all possible correct answers, and manual grading is used to review cases that were not properly graded.
+
+Instructors may specify a rubric to use positive or negative grading. In positive grading, submissions start at zero, and individual rubric items correspond to requirements to be added to the score. In negative grading, submissions start at the maximum score, and individual rubric items correspond to penalties subtracted from the score. Either option allows the creation of items with no points, which provide feedback that does not affect score. Additionally, rubric items may be set to a combination of positive and negative items, at the discretion of the grader.
+
+By default, the score produced by the rubric items is capped at the maximum points associated to the question, and penalties do not cause the points to get below zero. This limit can optionally be extended by assigning a minimum rubric score (typically to allow negative scores) and maximum extra credit to the rubric (to allow points above the maximum).
+
+Each rubric item is set a number of points and a description. This description is shown to both graders and in feedback to students. It is also possible to add an additional detailed explanation to the rubric item, which is shown to students upon request. Finally, a grader note can be set that is visible to graders, but not to students, and can be used to provide instructions on scenarios when the rubric should be used or not.
+
+Items can also be set to always be listed for students (default), or only if it is is selected. This option may be used in cases such as penalties that apply to a small subset of students, or for credit assigned to unconventional correct answers that are not to be advertised to students that did not use them.
+
+In the description, explanation, and grader note, it is possible to use mustache patterns based on the parameters, correct answers and submitted answers for a submission. This will cause the description of the item provided to graders and students to match the question variant assigned to the student, as shown in the screenshot below.
+
+![Rubric items in settings](rubric_items_setting.png)
+
+This will result in a grading panel containing a description with the value of the parameter replaced with the appropriate value for the student's variant.
```

Comment:
```suggestion
The teaching team will also see a preview of the rubric item description with the appropriate value for that student's submission.
```

I suggest swapping the order of what staff sees and what the student sees (since this is super cool - and most useful for students)